zetaclient/zetacore/client_monitor.go

```diff
@@ -17,8 +17,9 @@ import (
 
 // constants for monitoring tx results
 const (
-	monitorInterval   = constant.ZetaBlockTime
-	monitorRetryCount = 10
+	defaultMonitorTimeout = 2 * time.Minute
```

Comment:
default implies this is an option, but we just use this hardcoded value? So it's monitorTimeout

Should add a comment to explain the suage as well